app/controllers/admin/facets_controller.rb

```diff
@@ -35,8 +35,44 @@ def destroy
     redirect_to admin_facets_path
   end
 
+  def remove_item
+    begin
+      set_visibility
```

Comment:
I think we need less ambiguity - either the action (and private method) should have the word "toggle", or `set_visibility` should take a parameter and set visibility to that value.